src/views/MountainLayout/MountainLayoutView.tsx

```diff
@@ -20,14 +23,38 @@ const MountainLayoutView: FunctionComponent<Props> = ({data, width, height}) =>
         }))
     ), [data.views])
     const viewProps = useMemo(() => ({}), [])
-    return (
+    const content = (
         <MountainWorkspace
             viewPlugins={viewPlugins}
             viewProps={viewProps}
             width={width}
             height={height}
         />
     )
+    // const [sortingCuration, sortingCurationDispatch] = useReducer(sortingCurationReducer, {})
+    const {state: sortingCuration} = useSubfeedReducer({subfeedUri: data.sortingCurationUri}, sortingCurationReducer, {}, {actionField: false})
+    const sortingCurationDispatch = useCallback((a: SortingCurationAction) => {
+        if (!data.sortingCurationUri) return
+        initiateTask({
+          functionId: 'spikesortingview.sorting_curation_action.1',
+          kwargs: {
+            sorting_curation_uri: data.sortingCurationUri,
+            action: a
+          },
+          functionType: 'action',
+          onStatusChanged: () => {}
+        })
+        // this might be how we can do offline-first curation (get curationSubfeed from useSubfeedReducerS)
+        // curationSubfeed.appendOfflineMessages([a]) // this would need to be implemented
```

Comment:
So this would essentially be batching the changes locally and then pushing them all at once? Seems like that could be handy. We'd just need to define what happens if the remote feed status has updated before the batch is pushed--might require intelligent intervention on the user's part, but hopefully rare enough that it won't be too much of a problem.